src/main/scala/edu/arizona/sista/learning/RFClassifier.scala

```diff
@@ -628,7 +667,10 @@ case class Utility (feature:Int, // index of this feature in the feature lexicon
 
 class RFJob[L, F](
                    val dataset:CounterDataset[L, F],
-                   val indices:Array[Int],
+                   val trainIndices:Array[Int],
+                   val oobIndices:Array[Int],
+                   val activeNodes:Set[(Int, Double)],
+                   val nilLabel:Option[L],
                    val featureThresholds:Array[Array[Double]],
                    val currentUtility:Double,
                    val random:Random) {
```

Comment:
@marcovzla has been trying to get us all to follow these conventions: http://docs.scala-lang.org/style/indentation

What do you think?